### ISSUE TITLE
ImageMagick / Inkscape automatic install failed

### ISSUE THREAD

**Issue by AnselmD** [gnosygnu/xowa#33]
I think the reason is, there are no ImageMagick-*-Q??-windows.zip any more - only portable.

BTW, is there a direkt link from the XOWA main page to the xowa github issues page? I can not find one.

**Comment by gnosygnu** [comment]
Hey! Good to hear from you again!

> I think the reason is, there are no ImageMagick-*-Q??-windows.zip any more - only portable.

I thought I removed all links to ImageMagick and Inkscape. XOWA no longer needs either program unless it's for the mass thumbnail builder (command-line). Which page are you seeing the links on?

> BTW, is there a direkt link from the XOWA main page to the xowa github issues page? I can not find one.

Nope. I just added one now. It'll be part of the next release.

```
|-
| [https://github.com/gnosygnu/xowa/issues Report issues]
| Report bugs / issues on Github
```

**Comment by AnselmD** [comment]
>Hey! Good to hear from you again!
Nice to meet you again ;-)

>Which page are you seeing the links on?
home/wiki/Help:Contents
-> Other setup 	

    Setting up images: (Windows) (Linux) (Mac OS X) 
home/wiki/Help:Set_up_images/Windows

>Nope. I just added one now. It'll be part of the next release.
Oh, you did it at inside the applications. Good!

I missed it here:
XOWA: Main Page
https://gnosygnu.github.io/xowa/

and found it here:
XOWA: Feedback
https://gnosygnu.github.io/xowa/contact.html

**Comment by gnosygnu** [comment]
> home/wiki/Help:Set_up_images/Windows

Oops. I checked the page, but didn't see the "Automatic install" at the bottom.

It fails because of inkscape. They moved off sourceforge.net, and the url was still pointing there.

Using "Edit", I updated the url to the new download: https://inkscape.org/en/gallery/item/3932/Inkscape-0.91-1-win32.7z . However, this now fails with "could not generate dh keypair". This looks like an outdated Java issue: http://stackoverflow.com/questions/6851461/java-why-does-ssl-handshake-give-could-not-generate-dh-keypair-exception

I just ended up commenting it out for now.

> I missed it here...

Yup, but putting it on the Main Page gives it more visibility. Thanks for the suggestion!
